pynamodb/tests/test_attributes.py

```diff
@@ -723,6 +737,7 @@ def test_list_of_map_with_of(self):
         class Person(MapAttribute):
             name = UnicodeAttribute()
             age = NumberAttribute()
+            extra_arbitrary_data = JSONAttribute()
```

Comment:
Gotcha, makes sense. I think I'd prefer to see an individual test case for this to make it clear that's what we're testing for too, but this can stay as a higher-level test